### PR TITLE
Feat 7733

### PR DESCRIPTION
**Pull request by Poppe-A**
#7733

### CODE REVIEW

File: apps/catalog/catalog/src/app/landing/landing.component.scss [hunk 1]
```diff
@@ -138,6 +134,9 @@ landing-app-link {
     landing-content {
       .landing-presentation {
         mat-tab-group {
+          article .surface {
```
Comment (RemcoSimonides) [inline apps/catalog/catalog/src/app/landing/landing.component.scss:137]: Isn't this the same as only `.surface` ?

File: apps/catalog/catalog/src/app/landing/landing.component.scss [hunk 2]
```diff
@@ -42,32 +42,28 @@ catalog-landing {
...
-          width: 300px;
-          height: 200px;
-        }
-        div {
-          padding-inline: 40px;
-          flex: 1 1 0;
+        article {
+          margin-bottom: 24px;
+          .surface {
+            width: 32%;
```
Comment (RemcoSimonides) [inline apps/catalog/catalog/src/app/landing/landing.component.scss:52]: `.surface` is for making it look like a card without being a card. So you should target it's element here instead I think.
Shouldn't it be 33% if you want to have 3 of them in a row? Also, how does it work for mobile? Do you still want 3 in a row?

File: apps/catalog/catalog/src/app/landing/landing.component.scss [hunk 3]
```diff
@@ -42,32 +42,28 @@ catalog-landing {
...
+            text-align: center;
+            }
+        }
+      
+        .benefits {
+          > div {
```
[the remark this answers is not in the document]
Comment (Poppe-A) [reply apps/catalog/catalog/src/app/landing/landing.component.scss:58]: Yep, I don't want to apply this style to all the div, only the div which are direct child of `.benefits`

File: assets/images/light/content_buyer_benef1.svg [hunk 4]
```diff
@@ -0,0 +1,79 @@
+<svg width="300" height="149" viewBox="0 0 300 149" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
[the remark this answers is not in the document]
Comment (Poppe-A) [reply assets/images/light/content_buyer_benef1.svg:1]: Indeed !

File: apps/catalog/catalog/src/app/landing/landing.component.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (RemcoSimonides) [reply apps/catalog/catalog/src/app/landing/landing.component.scss:61]: Then give it a class name instead. I feel like you're the only one who understand what `&*(!@<!@N` does in css but I think most of us don't so let's make it human-readable.

**Review by RemcoSimonides — APPROVED** [review]
Remove the `benef` files. They are now added twice in your PR

**Pull request merged**